src/Components/WebAssembly/Sdk/integrationtests/WasmBuildLazyLoadTest.cs

```diff
@@ -165,6 +165,62 @@ public async Task Publish_LazyLoadExplicitAssembly_Release_Works()
             Assert.Contains("blazorwasm.dll", assemblies.Keys);
         }
 
+        [Fact]
+        public async Task Build_LazyLoadExplicitAssembly_InvalidAssembly()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("blazorwasm", additionalProjects: new[] { "razorclasslibrary" });
+            project.Configuration = "Release";
+
+            project.AddProjectFileContent(
+@"
+<ItemGroup>
+    <BlazorWebAssemblyLazyLoad Include='RazorClassLibraryInvalid.dll' />
+</ItemGroup>
+");
+            // Act
+            var result = await MSBuildProcessManager.DotnetMSBuild(project);
+
+            // Assert
+            Assert.BuildError(result, "BLAZORSDK1001");
+            Assert.BuildFailed(result);
+        }
+
+        [Fact]
+        public async Task Publish_LazyLoadExplicitAssembly_InvalidAssembly()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("blazorwasm", additionalProjects: new[] { "razorclasslibrary" });
+            project.Configuration = "Release";
+
+            project.AddProjectFileContent(
+@"
+<ItemGroup>
+    <BlazorWebAssemblyLazyLoad Include='RazorClassLibraryInvalid.dll' />
+</ItemGroup>
+");
+            // Act
+            var result = await MSBuildProcessManager.DotnetMSBuild(project, "Publish");
+
+            // Assert
+            Assert.BuildError(result, "BLAZORSDK1001");
+            Assert.BuildFailed(result);
+        }
+
+        [Fact]
+        public async Task Build_Without_LazyLoadExplicitAssembly_Works()
```

Comment:
I don't think this test is necessary. It's covered by our publish tests and we have coverage on a standard lazy-load scenario as well.

Reply:
Go it, I will change it